unity-wrapper/src/main/java/com/linecorp/linesdk/unitywrapper/CallbackPayload.kt

```diff
@@ -0,0 +1,31 @@
+package com.linecorp.linesdk.unitywrapper
+
+import com.google.gson.Gson
+import com.unity3d.player.UnityPlayer
+
+
+data class CallbackPayload(
```

Comment:
How about renaming `CallbackPayload` to `UnityCallback` and change its signature from `value: String` to `payload: Payload?`. 
```kotlin
data class UnityCallback(val identifier: String, val payload: Payload?) {

    fun sendMessageOk() =
        UnityPlayer.UnitySendMessage(
            KEY_LINE_SDK,
            NAME_API_OK,
            generatePayloadJson()
        )

    fun sendMessageError() =
        UnityPlayer.UnitySendMessage(
            KEY_LINE_SDK,
            NAME_API_ERROR,
            generatePayloadJson()
        )

    private fun generatePayloadJson(): String {
        val value = payload?.toJson(gson) ?: gson.toJson(JsonNull.INSTANCE)
        return CallbackPayload(identifier, value).toJson(gson)
    }

    companion object {
        private val gson: Gson = Gson()

        private const val KEY_LINE_SDK: String = "LineSDK"
        private const val NAME_API_OK: String = "OnApiOk"
        private const val NAME_API_ERROR: String = "OnApiError"
    }
}
```

And, we declare a interface `Payload` and another data class `data class CallbackPayload(val identifier: String, val value: String)`
```kotlin
data class CallbackPayload(
    val identifier: String,
    val value: String
) {
    fun toJson(gson: Gson): String = gson.toJson(this)
}

interface Payload {
    fun toJson(gson: Gson): String = gson.toJson(this)
}
```